### PR TITLE
Only require importlib_metadata for Python<3.8

### PR DESCRIPTION
**Pull request by mr-c**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
         'tzlocal',
         'requests',
         'certifi',
-        'importlib_metadata',
+        "importlib_metadata;python_version<'3.8'",
```
Comment (mikedarcy) [inline setup.py:56]: This is not legal syntax here, nor do I think it is required for the desired outcome. Probably should just revert this. 

What really should happen is the entire `requires` block needs to be removed, as it is basically deprecated and its use with `install_requires` is not recommended. This is really just a legacy appendage that needs pruning. I'd prefer it if you reverted it and then I will take out the entire block after merging.

File: setup.py [hunk 2]
```diff
@@ -63,7 +63,7 @@
                       'tzlocal',
                       'certifi',
                       'packaging',
-                      'importlib-metadata',
+                      "importlib_metadata;python_version<'3.8'",
```
Comment (mikedarcy) [inline setup.py:66]: The quoting can just be inverted to match the rest of the entries. For example:
```
'importlib_metadata;python_version<"3.8"'
```

**Review by mikedarcy — CHANGES_REQUESTED** [review]
Please see review comments.

File: setup.py (hunk 2 above)
Comment (mr-c) [reply setup.py]: Done!

**Comment by mr-c** [comment]
Thanks!

**Pull request merged**